.github/workflows/policy-check.yml

```
@@ -0,0 +1,28 @@
+name: Policy check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Check source code for policy violations
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install profanity-check
```

Let's pin the version

```suggestion
        pip install profanity-check==1.0.3
```

---

good point!